Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -320,10 +320,12 @@ type handlerEntry struct {
 type PrintOptions struct {
 	NoHeaders          bool
 	WithNamespace      bool
+	WithKind           bool
 	Wide               bool
 	ShowAll            bool
 	ShowLabels         bool
 	AbsoluteTimestamps bool
+	KindName           string
```

Comment:
`WithKind` feels like the other controls / print options, it tells the printer whether to show or not kind prefixes next to object names.
But `KindName` is really strange. The more I think about it, it seems to be just some resource-specific value hidden into the printer.

Why does a printer need to know about a kind name?! Isn't the printer generic and able to print resources of different kinds?!